src/plugins/intel_npu/tools/protopipe/src/parser/config.cpp

```diff
@@ -453,6 +519,9 @@ struct convert<Network> {
 template <>
 struct convert<CPUOp> {
     static bool decode(const Node& node, CPUOp& op) {
+        std::cout << "Septi: " << __FILE__ << ":" << __LINE__ << "   ---   " << __func__ << std::endl;
+        const std::unordered_set<std::string> parameters = {"time_in_us"};
```

Comment:
This node also contains all keys of `OpDesc` besides `time_in_us`

Reply:
Done